test/controllers/user.test.js

```diff
@@ -5,6 +5,10 @@ const bcrypt = require('bcrypt');
 const models = require('../../app/models/index');
 const app = require('../../app');
 const { factoryAllModels } = require('../factory/factory_by_models');
+const config = require('../../config');
+const constants = require('../../lib/constants');
+
+const { secret_key } = config.common.api;
```

Comment:
Constants (and variables) should never be in `snake_case`. For config constants like this one, `UPPER_CASE` can be ok, but be consistent. If you wanna use `UPPER_CASE`, make sure other config constant definitions are in `UPPER_CASE` too.